### DISCUSSION TITLE
密碼生成器可以生成詞組

### DISCUSSION THREAD

**Discussion by alctny** [he3-app/he3-feedback#61] [Ideas]
希望密碼生成器可以生成單詞組合，如同 [1password generator](https://1password.com/password-generator/) 的 Memorable 模式。

這樣生成的密碼會更容易記憶以及輸入

**Comment by hongchaodeng** [comment]
你好，你的图没有上传成功。能否重新上传一下？谢谢🙏

**Reply by alctny** [reply]
那是 1password 密碼生成器的官網連接，Markdown 不小心把多寫了一個歎號，已經修改。

**Reply by hongchaodeng** [reply]
明白了，感谢你的建议！我们先记一下哈
